### PR TITLE
Add Dynatrace output

### PR DESCRIPTION
**Pull request by blu3r4y**
**What type of PR is this?**

/kind feature

**Any specific area of the project related to this PR?**

/area outputs

**What this PR does / why we need it**:

Adds [Dynatrace](http://dynatrace.com/) output to falcosidekick.

### CODE REVIEW

**Comment by Issif** [comment]
I'm in PTO this week, I will review this later when I'm back. FYI, I released a new version few days ago, I don't know when the next one will be out. Do you need this integration asap?

**Comment by blu3r4y** [comment]
Hi @Issif, thanks for your quick response. We would like to announce this in a blog post within the next two weeks. Having a (pre-)release within the next couple of weeks would be awesome!! No hurry for now, enjoy your PTO, see you next week :)

**Comment by Issif** [comment]
@blu3r4y I'll do my best to have a 2.29.0-rc1 next week. This is a great news, if you need anything for your blog post, tell me, that's my job to help you. If you agree, you could also propose a post for falco.org or even better, participate in our use-case section.

File: README.md [hunk 1]
```diff
@@ -1204,6 +1210,10 @@ order is
   password is not empty (default: "")
 - **OPENOBSERVE_CUSTOMHEADERS** : a list of comma separated custom headers to add,
   syntax is "key:value,key:value"
+- **DYNATRACE_APITOKEN** : Dynatrace API token with the "logs.ingest" scope, more info : https://dt-url.net/8543sda
```
Comment (Issif) [inline README.md:1213]:
```suggestion
- **DYNATRACE_APITOKEN** : Dynatrace API token with the "logs.ingest" scope, more info : https://dt-url.net/8543sda, if not empty, Dynatrace output is enabled 
```

File: config_example.yaml [hunk 2]
```diff
@@ -475,4 +475,9 @@ openobserve:
   # username: "a" # use this username to authenticate to OpenObserve if the username is not empty (default: "")
   # password: "" # use this password to authenticate to OpenObserve if the password is not empty (default: "")
   # customHeaders: # Custom headers to add in POST, useful for Authentication
-  #   key: value
\ No newline at end of file
+  #   key: value
+
+dynatrace:
+  apitoken: "" # Dynatrace API token with the "logs.ingest" scope, more info : https://dt-url.net/8543sda
```
Comment (Issif) [inline config_example.yaml:481]:
```suggestion
  apitoken: "" # Dynatrace API token with the "logs.ingest" scope, more info : https://dt-url.net/8543sda, if not empty, Dynatrace output is enabled 
```

File: config_example.yaml [hunk 3]
```diff
@@ -475,4 +475,9 @@ openobserve:
...
+  apiurl: "" # Dynatrace API url, use https://ENVIRONMENTID.live.dynatrace.com/api for Dynatrace SaaS and https://YOURDOMAIN/e/ENVIRONMENTID/api for Dynatrace Managed, more info : https://dt-url.net/ej43qge
+  # minimumpriority: "" # minimum priority of event for using this output, order is emergency|alert|critical|error|warning|notice|informational|debug or "" (default)
```
Comment (Issif) [inline config_example.yaml:483]: the `checkcert ` is missing here but declared in the struct and at client creation

**Review by Issif — CHANGES_REQUESTED** [review]

**Comment by blu3r4y** [comment]
> @blu3r4y I'll do my best to have a 2.29.0-rc1 next week. This is a great news, if you need anything for your blog post, tell me, that's my job to help you. If you agree, you could also propose a post for falco.org or even better, participate in our use-case section.

@Issif Awesome, thanks! Also, we do plan additional content in that direction. I will make sure to reach out when we get there.

One specific question: We would love to use the Falco logo in the header for our blog post. As long as we comply with the CNCF brand guidelines this should be fine, right?

**Comment by Issif** [comment]
Thank you.

For the logo, feel free to use it as you want, you can find the assets here: https://falco.org/community/falco-brand/

**Review by Issif — APPROVED** [review]
:+1:

**Comment by Issif** [comment]
@blu3r4y Here's the release candidate https://github.com/falcosecurity/falcosidekick/releases/tag/2.29.0-rc.1

**Comment by blu3r4y** [comment]
@Issif This is awesome, thanks you so much for your swift assistance! 🥳

**Pull request merged**